### PR TITLE
Apply Embulk Gradle plugin and upgrade Gradle wrapper

### PR DESCRIPTION
**Pull request by huylenq**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -55,58 +49,31 @@ task checkstyle(type: Checkstyle) {
...
+    licenses = ['Apache-2.0']
+}
 
-  spec.add_development_dependency 'bundler', ['~> 1.0']
-  spec.add_development_dependency 'rake', ['~> 12.0']
-end
-/$)
-    }
+gemPush {
+    host = "https://treasuredata.jfrog.io/treasuredata/api/gems/rubygems-local"
```
Comment (minidragon88) [inline build.gradle]: This host is not valid. Please use `https://rubygems.org/`

**Review by minidragon88 — CHANGES_REQUESTED** [review]
Please help to delete file https://github.com/treasure-data/embulk-input-jira/blob/apply-embulk-gradle-plugin/lib/embulk/input/jira.rb

**Comment by huylenq** [comment]
> Please help to delete file https://github.com/treasure-data/embulk-input-jira/blob/apply-embulk-gradle-plugin/lib/embulk/input/jira.rb

Ha, I didn't realize this has it's own Ruby guess plugin. Simply removing it won't work. The guess plugin (`lib/embulk/guess/jira.rb`) has to be packed along on the final output gem. Thus we have to either:
- [A] Revert this PR and use the old Gradle configuration.
- [B] Or tinkling "org.embulk.embulk-plugins" Gradle plugin to see if there're anyway to bundle that guess plugin along. I honestly don't know whether this is supported or not.

**Comment by huylenq** [comment]
Hi @dmikurube, a question, is [B] possible?

**Comment by dmikurube** [comment]
@huylenq The Gradle plugin has a support 
* Not to use an auto-generated Ruby startup code, and
* To include an original Ruby (guess) startup code

**Comment by minidragon88** [comment]
> > Please help to delete file https://github.com/treasure-data/embulk-input-jira/blob/apply-embulk-gradle-plugin/lib/embulk/input/jira.rb
> 
> Ha, I didn't realize this has it's own Ruby guess plugin. Simply removing it won't work. The guess plugin (`lib/embulk/guess/jira.rb`) has to be packed along on the final output gem. Thus we have to either:
> 
> * [A] Revert this PR and use the old Gradle configuration.
> * [B] Or tinkling "org.embulk.embulk-plugins" Gradle plugin to see if there're anyway to bundle that guess plugin along. I honestly don't know this is supported or not.

As my testing, remove the https://github.com/treasure-data/embulk-input-jira/blob/apply-embulk-gradle-plugin/lib/embulk/input/jira.rb does not have any side effect, we still keep the guess plugin

**Comment by minidragon88** [comment]
> > > Please help to delete file https://github.com/treasure-data/embulk-input-jira/blob/apply-embulk-gradle-plugin/lib/embulk/input/jira.rb
> > 
> > 
> > Ha, I didn't realize this has it's own Ruby guess plugin. Simply removing it won't work. The guess plugin (`lib/embulk/guess/jira.rb`) has to be packed along on the final output gem. Thus we have to either:
> > 
> > * [A] Revert this PR and use the old Gradle configuration.
> > * [B] Or tinkling "org.embulk.embulk-plugins" Gradle plugin to see if there're anyway to bundle that guess plugin along. I honestly don't know this is supported or not.
> 
> As my testing, remove the https://github.com/treasure-data/embulk-input-jira/blob/apply-embulk-gradle-plugin/lib/embulk/input/jira.rb does not have any side effect, we still keep the guess plugin

Checked again, it wont work if missing https://github.com/treasure-data/embulk-input-jira/blob/apply-embulk-gradle-plugin/lib/embulk/input/jira.rb

**Review by minidragon88 — APPROVED** [review]
LGTM

File: build.gradle (hunk 1 above)
Comment (huylenq) [reply build.gradle]: My mistake, thanks!

**Pull request merged**